File: generated-sources/scala-gatling/mojang-authentication/src/gatling/scala/com/github/asyncmc/mojang/authentication/scala/gatling/model/AccessKeys.scala

package com.github.asyncmc.mojang.authentication.scala.gatling.model


case class AccessKeys (
    _accessToken: String,
    _clientToken: Option[String]
)
object AccessKeys {
    def toStringBody(var_accessToken: Object, var_clientToken: Object) =
        s"""
        | {
        | "accessToken":$var_accessToken,"clientToken":$var_clientToken
        | }
        """.stripMargin
}

File: generated-sources/kotlin-spring/mojang-authentication/src/main/kotlin/com/github/asyncmc/mojang/authentication/kotlin/spring/model/AccessKeys.kt
package com.github.asyncmc.mojang.authentication.kotlin.spring.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param accessToken 
 * @param clientToken 
 */
data class AccessKeys (

        @get:NotNull 
        @JsonProperty("accessToken") val accessToken: String,

        @JsonProperty("clientToken") val clientToken: String? = null
) {

}

